platform-api-client-gwt/che-core-client-gwt-git/src/main/java/org/eclipse/che/api/git/gwt/client/GitServiceClient.java

```diff
@@ -197,6 +198,22 @@ void remoteList(String workspaceId,
                     AsyncRequestCallback<List<Remote>> callback);
 
     /**
+     * Get the list of remote repositories for pointed by <code>workDir</code> parameter one.
```

Comment:
Plz, guys don't copy paste old javadocs. I don't see any **workDir** parameter in method signature